src/Serializer/AbstractItemNormalizer.php

```diff
@@ -789,9 +822,16 @@ protected function normalizeRelation(ApiProperty $propertyMetadata, ?object $rel
             return $normalizedRelatedObject;
         }
 
-        $iri = $this->iriConverter->getIriFromResource($relatedObject);
+        $context['iri'] = $iri = $this->iriConverter->getIriFromResource($relatedObject);
+
```

Comment:
```suggestion
```

Reply:
Fixed in https://github.com/api-platform/core/pull/5758/commits/828ecb60ffeab01c34c9b6f922b208832d3a9d3d